.github/workflows/ci.yml

```diff
@@ -26,6 +26,17 @@ jobs:
           args: --timeout=${{ env.GOLANGCI_LINT_TIMEOUT }}
       - name: Verify Go modules
         run: go mod verify
+      - name: Run Go import formatting
+        run: make go-import-fmt
+      - name: Detect git changes
+        run: |
+          if [[ $(git diff --stat) != '' ]]; then
+            echo -e '❌ \033[0;31m. Run 'make go-import-fmt'.\033[0m'
```

Comment:
or maybe `make lint-fix` ? just in case the `golangci-lint` or `go mod verify` will fail 🤔  However, we will need add `if: alwasy()` to  other checks, to ensure that the next step will be executed.

We can also add `go mod tidy` to `lint-fix` target 🤔